quantum/encoder.c

```diff
@@ -107,34 +110,33 @@ static bool encoder_update(uint8_t index, uint8_t state) {
 #ifdef SPLIT_KEYBOARD
     index += thisHand;
 #endif
-    encoder_pulses[i] += encoder_LUT[state & 0xF];
+
+    int8_t pulse = encoder_LUT[state];
+    encoder_pulses[i] += pulse;
     if (encoder_pulses[i] >= resolution) {
-        encoder_value[index]++;
-        changed = true;
+        encoder_pulses[i] -= resolution;
         encoder_update_kb(index, ENCODER_COUNTER_CLOCKWISE);
-    }
-    if (encoder_pulses[i] <= -resolution) {  // direction is arbitrary here, but this clockwise
-        encoder_value[index]--;
-        changed = true;
+    } else if (encoder_pulses[i] <= -resolution) {
+        encoder_pulses[i] += resolution;
         encoder_update_kb(index, ENCODER_CLOCKWISE);
+    } else {
+        return 0;
     }
-    encoder_pulses[i] %= resolution;
-#ifdef ENCODER_DEFAULT_POS
```

Comment:
Is this logic now encapsulated elsewhere?

Reply:
Question still unresolved, `ENCODER_DEFAULT_POS` functionality should either be reintroduced or docs updated